library/TabsWrapper.class.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * library/TabsWrapper.class.php
+ *
+ * Wrapper for implementing tabs. Currently based on jQuery UI Tabs.
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see
+ * http://www.gnu.org/licenses/licenses.html#GPL .
+ *
+ * @package OpenEMR
+ * @license http://www.gnu.org/licenses/licenses.html#GPL GNU GPL V3+
+ * @author  Rod Roark <rod@sunsetsystems.com>
+ * @link    http://www.open-emr.org
+ */
+
+class TabsWrapper
+{
+    public $tabsid;
+    public $tabs = array();
+
+    function __construct($tabsid = 'tabs')
+    {
+        $this->tabsid = $tabsid;
+    }
+
+    // Declare an initial tab that will not be dynamically created.
+    // Commonly this is called once but you can make as many as you want, or none.
+    //
+    public function declareInitialTab(
+        $title = 'Initial Tab', // Literals not translated here because they are just examples.
```

Comment:
Travis CI gives an error "Multi-line function declarations must define one parameter per line" here, but I don't see what the problem is.